cli/pkg/release/gbm.go

```diff
@@ -0,0 +1,148 @@
+package release
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/wordpress-mobile/gbm-cli/pkg/console"
+	"github.com/wordpress-mobile/gbm-cli/pkg/exec"
+	"github.com/wordpress-mobile/gbm-cli/pkg/gh"
+	g "github.com/wordpress-mobile/gbm-cli/pkg/git"
+	"github.com/wordpress-mobile/gbm-cli/pkg/render"
+	"github.com/wordpress-mobile/gbm-cli/pkg/repo"
+	"github.com/wordpress-mobile/gbm-cli/pkg/utils"
+)
+
+func CreateGbmPR(version, dir string) (gh.PullRequest, error) {
+	var pr gh.PullRequest
+
+	gbmDir := fmt.Sprintf("%s/gutenberg-mobile", dir)
+	git := g.NewClient(gbmDir, true)
+
+	// org, err := repo.GetOrg("gutenberg-mobile")
+	// console.ExitIfError(err)
+
+	branch := "release/" + version
+
+	console.Info("Checking if branch %s exists", branch)
+	exists, _ := gh.SearchBranch("gutenberg-mobile", branch)
+
+	if (exists != gh.Branch{}) {
+		console.Info("Branch %s already exists", branch)
+		return pr, nil
+	} else {
+		console.Info("Cloning Gutenberg Mobile to %s", dir)
+
+		err := git.Clone(repo.GetRepoPath("gutenberg-mobile"), "--depth=1")
```

Comment:
@derekblank  I made a similar change in `gb.go`
```suggestion
		err := git.Clone(repo.GetRepoPath("gutenberg-mobile"), "--depth=1", ".")
```
The problem with letting `git.Clone` cloning into the default directory (`gutenberg-mobile`) is that the git client is configured to the parent directory `dir`.

With this change we'll also have to replace  `gbmDir` on line 19 with `dir`

Reply:
Or the other option is to create a separate git client for cloning and another for all the other git operations. I thought cloning into `.` was cleaner for gb.go.